modules/m4d-kafka/Makefile

```diff
@@ -4,6 +4,7 @@ all: helm-all
 ROOT_DIR := ../..
 DOCKER_NAME := m4d-kafka
 DOCKER_TAGNAME := 0.1.0
+HELM_VALUES := --values values.yaml.sample
```

Comment:
Why is this needed? How can we test the values used by the blueprint controller to install this chart?

Reply:
How I understood this is that @hunchback uses these sample files for testing of the helm charts. Thus not in combination with blueprints.

Reply:
correct, these values are used for testing purposes